dashboard/app/controllers/report_abuse_controller.rb

```diff
@@ -93,4 +96,14 @@ def report_abuse_form
       age: (current_user.age unless current_user.nil?),
     }
   end
+
+  private
+
+  def check_if_featured(project_id)
+    _, channel_id = storage_decrypt_channel_id(project_id)
+    return render_404 unless channel_id
+    @featured_project = FeaturedProject.find_by storage_app_id: channel_id
+    return false unless @featured_project
+    @featured_project.featured?
```

Comment:
Two things here:

1. `@featured_project` creates a variable on the `ReportAbuseController` instance, when it looks like you meant to just create a local variable (it's only used in this method).  `featured_project` without the `@` is what you want.

2. Idiomatic Ruby tends to be much terser than JavaScript - I'd use the safe-navigation operator `&.` and replace these last three lines with
   ```rb
   FeaturedProject.find_by(storage_app_id: channel_id)&.featured?
   ```